src/components/sectioningConfirm/ConfirmLabware.tsx

```diff
@@ -108,7 +159,8 @@ const ConfirmLabware: React.FC<ConfirmLabwareProps> = ({
                   key={slot.address}
                   slot={slot}
                   value={addressToCommentMap.get(slot.address) ?? ""}
-                  disabled={current.matches("done")}
+                  disabledComment={current.matches("done")}
+                  disabledSectionNumber={autoMode}
```

Comment:
Previously the section number would be disabled if `current.matches("done")`, but now it will only be disabled in automode. Should this be `disabledSectionNumber={autoMode || current.matches("done")}` ?

Reply:
Yes, you are right.